tests/tests/integration_tests.rs

```diff
@@ -54,6 +54,20 @@ async fn ws_subscription_works() {
 	}
 }
 
+#[tokio::test]
+async fn ws_subscription_with_input_works() {
+	let server_addr = websocket_server_with_subscription().await;
+	let server_url = format!("ws://{}", server_addr);
+	let client = WsClientBuilder::default().build(&server_url).await.unwrap();
+	let mut add_one: Subscription<u64> =
+		client.subscribe("subscribe_add_one", vec![1.into()].into(), "unsubscribe_add_one").await.unwrap();
+
+	for _ in 0..10 {
```

Comment:
just show a point that the actual param doesn't get mutated.

Reply:
maybe 5 iterations is enough? lol